enhancements/proposed/20210618-tips/tips.md

```diff
@@ -0,0 +1,215 @@
+
+```
+minikube start --kubernetes-version=v1.19.0
+```
+``````
+
+To this:
+
+```markdown
+You can specify any Kubernetes version you want. For example:
+
+minikube start --kubernetes-version=v1.19.0
+```
+
+Then we can print out the tip:
+
+
+```go
+boxCfg := out.BoxConfig{
+	Config: box.Config{
+		Py: 1,
+		Px: 5,
+		TitlePos: "Top",
+		Type: "Round",
+		Color: tipBoxColor,
+	},
+	Title: tipTitle,
+	Icon: style.Tip,
+}
+
+out.BoxedWithConfig(boxCfg, tips.Tips[chosen] + "\n\n" + tipSuffix)
+```
+
+![Screenshot from 2021-06-18 00-58-02](https://user-images.githubusercontent.com/1311594/122508665-53bd6380-cfd0-11eb-9e99-a6c5935514d5.png)
+
+User can choose to disable this through `minikube config set disable-tips true`
+
+We will have `make generate-docs` generating the docs site based on this YAML file as well.
+
+We can have a `Nice to know` sub-page under `FAQ`?
+
+![Screenshot from 2021-06-18 01-00-30](https://user-images.githubusercontent.com/1311594/122508827-a139d080-cfd0-11eb-98bb-f7c3c1c604c2.png)
+
+
+### About the tip collection
+
+I plan to start with the command lines and cover almost all CLI usages of minikube.
+
+That includes but not limited to:
+- addons
+- cached images
+- command line completion
+- config
+- file copy
+- dashboard
+- delete minikube cluster
+- configure minikube's docker/podman env
+- image build / load / ls / rm
+- ip
+- logging
+- kubectl
+- mount file directory
+- multi-node
+- pause/unpause to save resource
+- multi-profile
+- surface URL to a k8s service
+- ssh into minikube
+- status
+- tunnel to connect to LB
+- update-check to check versions
+- update-context
+
+### Implementation
+
+I plan to open at least 4 PRs:
+
+1. `out.Boxed` with custom style
+2. random `tips` display with ability to disable through config, with an initial set of about 10 tips
+3. `make generate-docs` to sync tips to docs
+4. Add more tips
+
+## Alternatives Considered
+
+1. Is there a more preferred file format to YAML?
+
+2. Maybe we just want to sync the tips to the `FAQ` page list instead of creating a new page?
+
+3. Instead of the file format I proposed, maybe add a `question` field?
+
+    ```yaml
+    tips:
+      - question: How to specify a different Kubernetes version?
+        answer: |
+          You can specify any Kubernetes version you want. For example:
+
+          ```
+          minikube start --kubernetes-version=v1.19.0
+          ```
+      - question: Do I have to install `kubectl` myself?
+        answer: |
+          You can use minikube's built-in kubectl. For example:
+
+          ```
+          minikube kubectl -- get pods
+          ```
+      - question: How do I access the Kubernetes Dashboard UI?
+        answer: |   
+          minikube has the built-in Kubernetes Dashboard UI. To access it:
+
+          ```
+          minikube dashboard
+          ```
+    ```
+
+   On the docs side we should both questions and answers. On the CLI side
```

Comment:
Interesting idea! I think wording that correctly can be very tricky though, so it makes sense as a tip and as an answer. I'm a much bigger fan of just a set of tips.